iml-agent/src/crm_reader.rs

```diff
@@ -38,18 +38,10 @@ fn required_arg<'a>(arg: &str, x: &'a HashMap<&str, String>) -> Result<&'a str,
 }
 
 fn required_bool(arg: &str, x: &HashMap<&str, String>) -> Result<bool, ImlAgentError> {
-    required_arg(arg, x).and_then(str_to_bool)
-}
+    let x = required_arg(arg, x)?;
+    let x = x.parse()?;
```

Comment:
Could this just be:
```
x.parse()
```
Since it returns a result?

Reply:
No, because the errors wouldn't match. `?` is doing a conversion to `ImlAgentError`.

Reply:
👍